src/main/java/lotto/model/LottoTicket.java

```diff
@@ -0,0 +1,60 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTicket {
+    public static final Money SELLING_PRICE = new Money(1000);
+    public static final int LOTTO_SIZE = 6;
+    private final List<LottoNumber> numbers;
+
+    protected LottoTicket(List<LottoNumber> numbers) {
```

Comment:
```suggestion
    private LottoTicket(List<LottoNumber> numbers) {
```
상속을 없앴으니 접근제어자도 바꿔주는게 좋겠네요